packages/lambda-functions/functions/token/ensureTokenGenerated.ts

```diff
@@ -0,0 +1,71 @@
+require('isomorphic-fetch');
+require('abortcontroller-polyfill');
```

Comment:
This is needed for Platform client ?

Reply:
yes!

Reply:
Yeah, that's something that's nagging me everytime. I don't like this kind of 'script require' (i.e. require that modify the global scope) for NodeJS.

I plan to tackle that head-on when we have time ™️  (i.e. next hackathon).